tests/mr_test_module/src/lib.rs

```diff
@@ -887,8 +887,7 @@ static mut INT_RECORD_TYPE: Option<RecordType<IntRecord>> = None;
 fn init_func(ctx: &Context, _args: &Vec<RedisString>) -> Status {
```

Comment:
```suggestion
fn init_func(ctx: &Context, args: &Vec<RedisString>) -> Status {
```
`_` in rust means that argument is not in use, I see that now you use it so we can remove the `_`